docs/DeveloperGuide.md

```diff
@@ -7,32 +7,97 @@
 ## Design & implementation
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Basic Class Diagram 
+![image](https://user-images.githubusercontent.com/53790951/160271319-4a351f51-afd7-4e04-9451-f04143146551.png)
```

Comment:
Even if the class methods and fields are not represented, there should be another line separating the contents of the class.

<img width="220" alt="Screenshot 2022-03-31 at 11 39 38 AM" src="https://user-images.githubusercontent.com/49554923/160971756-9cabc8a5-0db1-4731-9581-c1d2080acbc1.png">